PICMI_Python/setup.py

```diff
@@ -9,7 +9,7 @@
     install_requires = [ line.strip('\n') for line in f.readlines() ]
 
 setup(name = 'picmistandard',
-      version = '0.0.21',
+      version = '0.0.22',
```

Comment:
The version update should be done in a commit separately from this PR. See `PICMI_Python/README.md` for how to update the version.

Reply:
I saw the README direction saying "it is ok to push a version update directly to the main branch", but I didn't understand that it **should** be done on the main branch. Now I know for next time :)